third_party/2and3/google/protobuf/internal/containers.pyi

```diff
@@ -9,8 +9,8 @@ from typing import (
     Iterable,
     Iterator,
     List,
-    Mapping as tMapping,
-    MutableMapping as tMutableMapping,
+    Mapping,
+    MutableMapping,
```

Comment:
@nipunn1313 Sorry for not being clearer, but this must explicitly be:

```suggestion
    Mapping as Mapping,
    MutableMapping as MutableMapping,
```

Basically, everything that has an 'as ...' clause gets reexported, even if the as clause is identical to the original name.

Reply:
ah I misunderstood - rereading your original comment I see it was there. Thanks for clarifying again. This makes sense. Good feature of the PEP.